package.json

```diff
@@ -4,18 +4,19 @@
   "description": "Compiles markdown files to html using remarkable",
   "version": "1.0.0",
   "keywords": ["grunt","markdown","remarkable"],
-  "homepage":"https://github.com/rwky/grunt-remarkable",
+  "homepage": "https://github.com/rwky/grunt-remarkable",
   "bugs": { "url": "https://github.com/rwky/grunt-remarkable/issues" },
   "engines": {
     "node": ">=0.10"
   },
   "dependencies": {
+    "async": "2.0.0-rc.5",
```

Comment:
Latest stable version of **async** seems to be "1.5.2"

https://github.com/caolan/async/releases